lib/ansible/modules/cloud/azure/azure_rm_sqldatabase.py

```diff
@@ -0,0 +1,576 @@
+    requested_service_objective_id:
+        description:
+            - "The configured service level objective ID of the database. This is the service level objective that is in the process of being applied to the
+               database. Once successfully updated, it will match the value of currentServiceObjectiveId property. If requestedServiceObjectiveId and request
+               edServiceObjectiveName are both updated, the value of requestedServiceObjectiveId overrides the value of requestedServiceObjectiveName. To see
+                possible values, query the capabilities API (/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationID}/capabilities) ref
+               erred to by operationId: 'Capabilities_ListByLocation.'"
+    requested_service_objective_name:
+        description:
+            - "The name of the configured service level objective of the database. This is the service level objective that is in the process of being applie
+               d to the database. Once successfully updated, it will match the value of serviceLevelObjective property. To see possible values, query the cap
+               abilities API (/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationID}/capabilities) referred to by operationId: 'Capab
+               ilities_ListByLocation.'."
+        choices:
+            - 'basic'
+            - 's0'
+            - 's1'
+            - 's2'
+            - 's3'
+            - 'p1'
+            - 'p2'
+            - 'p3'
+            - 'p4'
+            - 'p6'
+            - 'p11'
+            - 'p15'
+            - 'system'
+            - 'system2'
+            - 'elastic_pool'
+    elastic_pool_name:
+        description:
+            - "The name of the elastic pool the database is in. If elasticPoolName and requestedServiceObjectiveName are both updated, the value of requested
+               ServiceObjectiveName is ignored. Not supported for DataWarehouse edition."
+    read_scale:
+        description:
+            - "Conditional. If the database is a geo-secondary, readScale indicates whether read-only connections are allowed to this database or not. Not su
+               pported for DataWarehouse edition."
+        choices:
+            - 'enabled'
+            - 'disabled'
+    sample_name:
+        description:
+            - "Indicates the name of the sample schema to apply when creating this database. If createMode is not Default, this value is ignored. Not support
+               ed for DataWarehouse edition."
+        choices:
+            - 'adventure_works_lt'
+    zone_redundant:
+        description:
+            - Whether or not this database is zone redundant, which means the replicas of this database will be spread across multiple availability zones.
+
+extends_documentation_fragment:
+    - azure
+
+author:
+    - "Zim Kalinowski (@zikalino)"
+
+'''
+
+EXAMPLES = '''
+  - name: Create (or update) SQL Database
```

Comment:
Because there are a lot of options for this module, having 2 or more examples would be quite helpful for the docs.